<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Фильтр:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="imageFilterType"/>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Обработка краёв:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="borderProcessType"/>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="processButton">
      <property name="text">
       <string>Обработать</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="resetButton">
      <property name="text">
       <string>Сбросить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>processButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>changeImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>resetImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeImage()</slot>
  <slot>resetImage()</slot>
 </slots>
</ui>
